src/node_internals.h

```diff
@@ -538,6 +538,8 @@ int ThreadPoolWork::CancelWork() {
   return uv_cancel(reinterpret_cast<uv_req_t*>(&work_req_));
 }
 
+tracing::AgentWriterHandle* GetTracingAgentWriter();
```

Comment:
Is possible to put this declaration in a smaller scope (file with less dependance, maybe `node_platform.h`)? I'm worried that adding
https://github.com/nodejs/node/blob/cecac2badafb14dd9236913841114387a4fa4eac/src/inspector/tracing_agent.cc#L2
will complicate the dependency graph (hence the build, embedding, etc.)